### PR TITLE
re writing ingest to parquet as a proper transform

### PR DESCRIPTION
**Pull request by blublinsky**
## Why are these changes needed?

Show how to use binary transforms

### CODE REVIEW

**Comment by blublinsky** [comment]
As per agreement with @roytman, the kfp is prepared but not tested yet

**Review by daw3rd — CHANGES_REQUESTED** [review]
This will need wait on PR #213

File: transforms/code/ingest_2_parquet/kfp_ray/v1/ingest_2_parquet_wf.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    '"image_pull_secret": "", "image": "' + task_image + '"}',
+    server_url: str = "http://kuberay-apiserver-service.kuberay.svc.cluster.local:8888",
+    # data access
+    data_s3_config: str = "{'input_folder': 'test/ingest2parquet/input', 'output_folder': 'test/ingest2parquet/output/'}",
+    data_s3_access_secret: str = "s3-secret",
+    data_max_files: int = -1,
+    data_num_samples: int = -1,
+    data_files_to_use: str = "['.zip']",
+    # orchestrator
+    runtime_actor_options: str = f"{{'num_cpus': 0.8, `memory`: {2*GB}}}",
```
Comment (Mohammad-nassar10) [inline transforms/code/ingest_2_parquet/kfp_ray/v1/ingest_2_parquet_wf.py]: typo in the quotes of `memory`. It should be `'memory'`.

Comment (blublinsky) [reply transforms/code/ingest_2_parquet/kfp_ray/v1/ingest_2_parquet_wf.py]: Fixed

**Pull request closed without merging**